scripts/entrypoint.sh

```diff
@@ -12,16 +10,16 @@ if [ $# -eq 0 ] ; then
   sh ./scripts/start_server.sh
 elif [ "${1}" = "test" ] ; then
   echo "Run Tests"
-  make test
+  sh ./test/run_tests.sh
 elif [ "${1}" = "async" ] ; then
   sh ./scripts/run_async.sh
-elif [ "${1}" = "init" ] ; then
-  echo "Initialize module"
 elif [ "${1}" = "bash" ] ; then
-  bash
+  echo "This image only supports sh shells"
+  sh
+elif [ "${1}" = "sh" ] ; then
+  sh
 elif [ "${1}" = "report" ] ; then
-  export KB_SDK_COMPILE_REPORT_FILE=./work/compile_report.json
-  make compile
+  echo "The compile report can be found at ./work/compile_report.json"
```

Comment:
remove the compilation step as it's only used to generate the compile report file, which is already present in the repo.